### PR TITLE
Added example for :class:`Prism`

### PR DESCRIPTION
**Pull request by jojo96**
## Changelog / Overview

Added documentation for the prism class. Added an example since there were no existing examples for the prism class

## Motivation

Added example for new users

## Explanation for Changes

Makes it more readable as the examples are useful to the newer users

[Documentation Reference](https://manimce--1626.org.readthedocs.build/en/1626/reference/manim.mobject.three_dimensions.Prism.html)

## Checklist
- [x] I have read the [Contributing Guidelines](https://docs.manim.community/en/latest/contributing.html)
- [ ] I have written a descriptive PR title (see top of PR template for examples)
- [ ] I have written a changelog entry for the PR or deem it unnecessary
- [ ] My new functions/classes either have a docstring or are private
- [ ] My new functions/classes have [tests](https://github.com/ManimCommunity/manim/wiki/Testing) added and (optional) examples in the docs
- [x] My new documentation builds, looks correctly formatted, and adds no additional build warnings

## Reviewer Checklist
- [ ] The PR title is descriptive enough
- [ ] The PR is labeled correctly
- [ ] The changelog entry is completed if necessary
- [ ] Newly added functions/classes either have a docstring or are private
- [ ] Newly added functions/classes have [tests](https://github.com/ManimCommunity/manim/wiki/Testing) added and (optional) examples in the docs
- [ ] Newly added documentation builds, looks correctly formatted, and adds no additional build warnings

### CODE REVIEW

**Comment by kolibril13** [comment]
Thanks for this pull request! :)
Here is the example:
https://manimce--1626.org.readthedocs.build/en/1626/reference/manim.mobject.three_dimensions.Prism.html?highlight=prism
As it is a last frame example, I would add the prisms with `self.add(prismsmall)` and put them next to each other. No self.play calls are needed here.
I would have expected a prism to look more like this:
https://www.google.com/search?q=Prism&sxsrf=ALeKk013R6pIjgop-QuiqAymHDRhbRpt2w:1622901262537&source=lnms&tbm=isch&sa=X&ved=2ahUKEwj6zIiq0oDxAhUiJcUKHTi6C2oQ_AUoAXoECAEQAw&biw=1639&bih=946
As your example looks very cube-like, can you maybe find an example that looks more like a prism? :)

**Comment by Darylgolden** [comment]
> As your example looks very cube-like, can you maybe find an example that looks more like a prism? :)
That's a problem with the class, not this example. It really is just a cuboid.

**Review by behackl — DISMISSED** [review]
Thank you for your contribution! I agree with @kolibril13's suggestion: it would be better for the example if the small and large prisms were shifted and rendered next to each other in an image; and given that this is rendered as an image, the play and wait calls should then be removed.

Could you please adapt the example?

**Comment by jojo96** [comment]
> Thank you for your contribution! I agree with @kolibril13's suggestion: it would be better for the example if the small and large prisms were shifted and rendered next to each other in an image; and given that this is rendered as an image, the play and wait calls should then be removed.
> 
> Could you please adapt the example?

@behackl I have tried to implement the solution.

File: manim/mobject/three_dimensions.py [hunk 1]
```diff
@@ -260,6 +260,21 @@ def generate_points(self):
...
 class Prism(Cube):
+    """A Prism.
+
+    Examples
+    --------
+
+    .. manim:: ExamplePrism
+        :save_last_frame:
+
+        class ExamplePrism(Scene):
```
Comment (kolibril13) [inline manim/mobject/three_dimensions.py:271]:
```suggestion
        class ExamplePrism(ThreeDScene):
```

**Review by Darylgolden — CHANGES_REQUESTED** [review]
In addition, please also set a suitable camera orientation so that the 3D-ness of the prism can be seen, such as in the example [here](https://docs.manim.community/en/stable/examples.html#fixedinframemobjecttest)

File: manim/mobject/three_dimensions.py (hunk 1 above)
Comment (jojo96) [reply manim/mobject/three_dimensions.py]: Applied the changes

**Comment by jojo96** [comment]
> In addition, please also set a suitable camera orientation so that the 3D-ness of the prism can be seen, such as in the example [here](https://docs.manim.community/en/stable/examples.html#fixedinframemobjecttest)

@Darylgolden changed camera orientation

File: manim/mobject/three_dimensions.py [hunk 2]
```diff
@@ -284,6 +284,22 @@ def generate_points(self):
...
+
+    .. manim:: ExamplePrism
+        :save_last_frame:
+
+        class ExamplePrism(ThreeDScene):
+            def construct(self):
+                self.set_camera_orientation(phi=5 * DEGREES, theta=25 * DEGREES)
+                prismSmall = Prism(dimensions=[1,2,3]).rotate(PI/2)
+                prismLarge = Prism(dimensions=[1.5,3,4.5]).move_to([2,0,0])
+                self.add(prismSmall,prismLarge)
```
Comment (Darylgolden) [inline manim/mobject/three_dimensions.py:300]:
```suggestion
    .. manim:: ExamplePrism
        :save_last_frame:

        class ExamplePrism(ThreeDScene):
            def construct(self):
                self.set_camera_orientation(phi=60 * DEGREES, theta=150 * DEGREES)
                prismSmall = Prism(dimensions=[1, 2, 3]).rotate(PI / 2)
                prismLarge = Prism(dimensions=[1.5, 3, 4.5]).move_to([2, 0, 0])
                self.add(prismSmall, prismLarge)
```

**Review by Darylgolden — CHANGES_REQUESTED** [review]
Adjusted angle and applied black formatting

**Review by Darylgolden — APPROVED** [review]
LGTM!

**Pull request merged**